File: trailing_zeroes.py
def trailing_zeroes():
    n = int(input().strip())

    factor_5_count = 0
    divisor = 5

    while divisor <= n:
        factor_5_count += n // divisor
        divisor *= 5

    print(factor_5_count)

# Example usage
trailing_zeroes()

File: trailing_zeroes.go
package main

import (
	"fmt"

)

func trailing_zeroes() {
	var n int
	fmt.Scanf("%d\n", &n)

	factor_5_count := 0
	divisor := 5

	for divisor <= n {
		factor_5_count += n/divisor
		divisor *= 5
	}

	fmt.Println(factor_5_count)
}
